modules/tinymce/src/plugins/table/main/ts/ui/Buttons.ts

```diff
@@ -239,6 +239,15 @@ const addButtons = (editor: Editor, selectionTargets: SelectionTargets, clipboar
     ),
     onSetup: selectionTargets.onSetupCellOrRow
   });
+
+  editor.ui.registry.addToggleButton('tablecaption', {
+    tooltip: 'Table caption',
+    onAction: () => {
+      editor.execCommand('mceTableToggleCaption');
+    },
```

Comment:
There's already a helper here to do this, so please reuse it.
```suggestion
    onAction: cmd('mceTableToggleCaption'),
```